drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -37,6 +37,18 @@ def client_search(input:, **args)
       resolve_clients(search_scope, **args)
     end
 
+    clients_field :client_omni_search, 'Client omnisearch' do |field|
+      field.argument :input, Types::HmisSchema::ClientSearchInput, required: true
+    end
+
+    def client_omni_search(input:, **args)
```

Comment:
Untested, but this would be my suggestion. I think it is a bit confusing as an API for this query to accept the full clean search inputs, when it really only is meant to use the text search.

```
    clients_field :client_omni_search, 'Client omnisearch' do |field|
      field.argument :text_search, String, 'Omnisearch string', required: true
    end

    def client_omni_search(text_search:, **args)
      client_scope = Hmis::Hud::Client.searchable_to(current_user).matching_search_term(text_search).
        joins(:enrollments).
        merge(Hmis::Hud::Enrollment.open_during_range((Date.today - 1.month)..Date.today)).
        order(date_updated: :desc)
      resolve_clients(client_scope, **args)
    end
```